srcpkgs/sequoia/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'sequoia'
+pkgname=sequoia
+version=0.16.0
+revision="1"
+wrksrc="${pkgname}-v${version}"
+build_style="gnu-makefile"
+build_helper="rust"
+make_install_args="PYTHON=disable"
+make_build_args="PYTHON=disable"
+make_build_target="build-release"
+hostmakedepends="make pkg-config clang rust cargo git"
```

Comment:
You can put just `cargo`, since it depends on `rust`.

`make` is available for all builds by default.

Why does the build require git? Consider pulling in versioned tarballs for each git submodule, if any.